wiki/Tournaments/ATT/2021/en.md

```diff
@@ -98,6 +123,84 @@ The Ausu!Taiko Tournament 2021 is run by various community members.
+| ConieSan ![][flag_TH] | 1 | **4** | ![][flag_PH] qish | [#1](https://osu.ppy.sh/community/matches/81657906) |
+| Beat43210 ![][flag_AU] | -1 | **0** | ![][flag_NZ] **arley\_** | *win by default* |
+| **Tsubasa2** ![][flag_AU] | **0** | -1 | ![][flag_PH] Astrid Park | *win by default* |
+| Bedwyr Aorta ![][flag_MY] | 3 | **4** | ![][flag_MY] **nikolpai\_15** | [#1](https://osu.ppy.sh/community/matches/81659814) |
+| **Aiery** ![][flag_PH] | **4** | 0 | ![][flag_MY] CrabCow | [#1](https://osu.ppy.sh/community/matches/81660028) |
+| **Ponamis** ![][flag_TH] | **4** | 0 | ![][flag_SG] dantoh000 | [#1](https://osu.ppy.sh/community/matches/81660953) |
+| **Naverlyn** ![][flag_ID] | **4** | 0 | ![][flag_MY] TaikOseania | [#1](https://osu.ppy.sh/community/matches/81661153) |
+| **Jerry** ![][flag_MY] | **4** | 0 | ![][flag_TH] KappuChinooo | [#1](https://osu.ppy.sh/community/matches/81662208) |
+| **Prehistoria** ![][flag_SG] | **0** | -1 | ![][flag_NZ] arley\_ | *win by default* |
+| **Kotaro** ![][flag_PH] | **4** | 0 | ![][flag_NZ] Blujae | [#1](https://osu.ppy.sh/community/matches/81662982) |
+| Tsubasa2 ![][flag_AU] | 3 | **4** | ![][flag_PH] **CoEqual** | [#1](https://osu.ppy.sh/community/matches/81663047) |
+| **ezeenS** ![][flag_PH] | **4** | 2 | ![][flag_PH] DescriptiCringe | [#1](https://osu.ppy.sh/community/matches/81664069) |
+| **Element118** ![][flag_SG] | **4** | 0 | ![][flag_AU] AmateurMonkeyYT | [#1](https://osu.ppy.sh/community/matches/81664398) |
+| **tzechi** ![][flag_SG] | **4** | 0 | ![][flag_TH] ConieSan | [#1](https://osu.ppy.sh/community/matches/81665397) |
+| **Prehistoria** ![][flag_SG] | **0** | -1 | ![][flag_AU] Beat43210 | *win by default* |
+| **jmeh07** ![][flag_PH] | 0 | -1 | ![][flag_SG] Bobfree | *win by default* |
+| **Element118** ![][flag_SG] | **4** | 3 | ![][flag_SG] dantoh000 | [#1](https://osu.ppy.sh/community/matches/81667577) |
+| **tzechi** ![][flag_SG] | **4** | 0 | ![][flag_MY] TaikOseania | [#1](https://osu.ppy.sh/community/matches/81667594) |
+| **Prehistoria** ![][flag_SG] | **0** | -1 | ![][flag_SG] \_gt | *win by default* |
+
+Sunday, 25 April 2021:
+
+| Player 1 |  |  | Player 2 | Match link |
+| --: | :-: | :-: | :-- | :-- |
+| **Katdon\_donKat** ![][flag_ID] | **0** | -1 | ![][flag_PH] Kotaro | *win by default* |
+| **Naverlyn** ![][flag_ID] | **4** | 0 | ![][flag_TH] ConieSan | [#1](https://osu.ppy.sh/community/matches/81682418) |
+| **Aiery** ![][flag_PH] | **4** | 0 | ![][flag_TH] KappuChinooo | [#1](https://osu.ppy.sh/community/matches/81682450) |
+| r1chyy ![][flag_AU] | 0 | **4** | ![][flag_ID] **Katdon\_donKat** | [#1](https://osu.ppy.sh/community/matches/81681284) |
+| **Katdon\_donKat** | **4** | 0 | ![][flag_NZ] Blujae | [#1](https://osu.ppy.sh/community/matches/81732057) |
+| **Jerry** ![][flag_MY] | **4** | ![][flag_MY] CrabCow | [#1](https://osu.ppy.sh/community/matches/81742310) |
```

Comment:
this should be **Jerry 5:2 CrabCow**, no?

Reply:
Suppose to be 4:1 according to the Challonge. I'll fill it.